Source: written by Pierre Lejeune, from Exherbo
Upstream: submitted, http://code.google.com/p/eid-mw/issues/detail?id=98
Reason: Fix PKCS11 LOG_LEVEL constants usage

--- a/pkcs11/src/log.c	2012-04-02 21:05:50.840503851 +0200
+++ b/pkcs11/src/log.c	2012-04-29 23:05:16.332253301 +0200
@@ -55,9 +55,9 @@
  *
  ******************************************************************************/
 #ifdef DEBUG
-unsigned int g_uiLogLevel          = LOG_LEVEL_SPY;
+unsigned int g_uiLogLevel          = LOG_LEVEL_PKCS11_SPY;
 #else
-unsigned int g_uiLogLevel          = LOG_LEVEL_WARNING;
+unsigned int g_uiLogLevel          = LOG_LEVEL_PKCS11_WARNING;
 #endif
 
 void *logmutex = NULL;
@@ -111,17 +111,17 @@
     switch (string[0])
     {
       case 'I':
-        if (level  < LOG_LEVEL_INFO) 
+        if (level  < LOG_LEVEL_PKCS11_INFO) 
           return;
         break;
 
       case 'S':
-        if (level  < LOG_LEVEL_SPY) 
+        if (level  < LOG_LEVEL_PKCS11_SPY) 
           return;
         break;
 
       case 'W':
-        if (level < LOG_LEVEL_WARNING) 
+        if (level < LOG_LEVEL_PKCS11_WARNING) 
           return;
         break;
 
@@ -215,12 +215,12 @@
     switch (string[0])
     {
       case 'I':
-        if (level  < LOG_LEVEL_INFO) 
+        if (level  < LOG_LEVEL_PKCS11_INFO) 
           return;
         break;
 
       case 'W':
-        if (level < LOG_LEVEL_WARNING) 
+        if (level < LOG_LEVEL_PKCS11_WARNING) 
           return;
         break;
 
@@ -371,17 +371,17 @@
     switch (string[0])
       {
       case 'I':
-        if (level  < LOG_LEVEL_INFO) 
+        if (level  < LOG_LEVEL_PKCS11_INFO) 
           return;
         break;
 
       case 'S':
-        if (level  < LOG_LEVEL_SPY) 
+        if (level  < LOG_LEVEL_PKCS11_SPY) 
           return;
         break;
 
       case 'W':
-        if (level < LOG_LEVEL_WARNING) 
+        if (level < LOG_LEVEL_PKCS11_WARNING) 
           return;
         break;
 
@@ -396,7 +396,7 @@
   //   return;
 
 //log template only in INFO mode
-//  if (level  < LOG_LEVEL_INFO) 
+//  if (level  < LOG_LEVEL_PKCS11_INFO) 
 //     return;
 
   log_trace(string, "size = %d", count);
